modules/net-lb-int/main.tf

```diff
@@ -54,7 +54,7 @@ resource "google_compute_region_backend_service" "default" {
   name                            = var.name
   description                     = var.description
   load_balancing_scheme           = "INTERNAL"
-  protocol                        = var.protocol
+  protocol                        = var.backend_service_config.protocol
```

Comment:
Can we just specify here `UNSPECIFIED` as per [docs](https://cloud.google.com/load-balancing/docs/internal#forwarding-rule-protocols) always?

We need anyway to have agreement between protocols here, so I'm reluctant to extend interface scope if the value has to match the other value that user is providing. 

If `UNSPECIFIED` will not work, then maybe just still use `var.protocol`, but in case its value is `L3_DEFAULT` replace that with `UNSPECIFIED`?

Reply:
Yep. I'd go with the second option to facilitate life of users.

Reply:
Actually, let me take it back after double checking the docs. Let's modify it directly to UNSPECIFIED. Good point. thanks!